src/helpers/duplicateRequestPreventor.ts

```diff
@@ -0,0 +1,39 @@
+import type { Request, Response, NextFunction } from 'express';
+import { sendError } from './utils';
+import redisClient from './redis';
```

Comment:
Why we are using redis ? other services don't use Redis. unlike other services, we use single instance for sequencer no?

Reply:
Don’t remember the exact reason, maybe sequencer was running on multiple instances then

Reply:
Refactor to remove Redis dependency.

I think Redis was for better access to the duplicator queue from the tests, which are now refactored as integration test instead of e2e